h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java

```diff
@@ -32,27 +33,27 @@
 
     
     // Metalearner algorithm
-    @API(level = API.Level.critical, direction = API.Direction.INOUT, gridable = true,
+    @API(level = API.Level.critical, direction = API.Direction.INOUT, 
             values = {"AUTO", "glm", "gbm", "drf", "deeplearning"},
             help = "Type of algorithm to use as the metalearner. " +
                     "Options include 'AUTO' (GLM with non negative weights; if validation_frame is present, a lambda search is performed), 'glm' (GLM with default parameters), 'gbm' (GBM with default parameters), " +
                     "'drf' (Random Forest with default parameters), or 'deeplearning' (Deep Learning with default parameters).")
     public StackedEnsembleModel.StackedEnsembleParameters.MetalearnerAlgorithm metalearner_algorithm;
 
     // For ensemble metalearner cross-validation
-    @API(level = API.Level.critical, direction = API.Direction.INOUT, gridable = true,
```

Comment:
We talked about turning off `griddable` for some parameters (namely, `metalearner_algorithm` and `metalearner_params`) since we will handle those in a [specialized fashion](https://0xdata.atlassian.net/browse/PUBDEV-5281) in the future.  However, I think we can still support `griddable = true` for `metalerner_nfolds`.  Can you see any problems with doing that?

Reply:
I was thinking to turn of `griddable` for all params since SE is not really supported in Grid yet.

Reply:
Turns out that the other H2O algos don't have `gridable = true` for `nfolds` or `fold_assignment`, so we should not turn that on here either.  It's good how it is.